app/main.py

```diff
@@ -85,6 +93,30 @@ def create_app() -> FastAPI:
     return app
 
 
+def get_auth_deps() -> list[Depends]:
+    "function in order to fetch dependencies for authentication"
+    deps = os.environ.get("OTEAPI_AUTH_DEPS")
+    if deps:
+        mods = [
+            module.replace(" ", str()).rpartition(".") for module in deps.split("|")
+        ]
+        try:
+            imports = [
+                getattr(import_module(module), classname)
+                for (module, dot, classname) in mods
+            ]
+        except Exception as error:
+            raise error
```

Comment:
This is the same as just not using a try/except - but I guess your intention here is to figure out what specific exceptions to catch and raise another specific exception with a custom message?